### ISSUE TITLE
Emmet suggestions bug in SASS files 

### ISSUE THREAD

**Issue by saminton** [microsoft/vscode#115854]
Issue Type: <b>Bug</b>

Since the latest update, I am seeing really weird Emmet suggestions in SASS files (not SCSS) with a higher priority than my SASS suggestions. For example I have got used to writing `background-color` with `bkco`. But now if I hit tab with `bkco` it will give me `bkco: ` rather than the expected `background-color: `like it used to.

![Screenshot 2021-02-05 at 11 41 54](https://user-images.githubusercontent.com/15970757/107023631-2ef5eb80-67a7-11eb-98e8-563549881329.png)

VS Code version: Code 1.53.0 (8490d3dde47c57ba65ec40dd192d014fd2113496, 2021-02-03T16:01:04.046Z)
OS version: Darwin x64 19.4.0

<details><summary>Extensions (15)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-css-formatter|aes|1.0.1
project-manager|ale|12.0.1
save-as-snippet|art|0.0.1
vscode-icontheme-nomo-dark|be5|1.3.6
glsl-canvas|cir|0.2.11
vs-code-xml-format|fab|0.1.5
discord-vscode|icr|4.1.0
vscode-peacock|joh|3.9.1
vsliveshare|ms-|1.0.3629
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
vscode-extension-auto-import|Nuc|1.4.3
shader|sle|1.1.4
sass-indented|syl|1.8.15
better-align|wwm|1.1.6

</details>

**Comment by rzhao271** [comment]
The current Emmet snippet for background-color is `bgc`. `bkco` definitely shouldn't be suggested, though. I also confirmed that I'm only able to reproduce the issue with SASS files.

Also, I just realized the other suggestions that show up aren't part of Emmet. However, I don't see the other suggestions (such as background-color) when I try typing in `bkco` into a SASS file, even though I do see them on CSS and SCSS files. Wondering if I have a setting misconfigured.
